protocol/communities/community.go

```diff
@@ -990,6 +1021,10 @@ func (o *Community) UpdateCommunityDescription(description *protobuf.CommunityDe
 	o.config.CommunityDescription = description
 	o.config.CommunityDescriptionProtocolMessage = rawMessage
 
+	if newControlNode != nil {
+		o.setControlNode(newControlNode)
+	}
+
```

Comment:
@osmaczko please, take a look in this [commit](https://github.com/status-im/status-go/pull/4187/commits/59dd24f19abc0aa99131deaa8b11a2e95cdc99e5) changes

Previously, `ControlNodeChanged` was not set correctly from the loop as we updated the original `Community` ,while we need to update the modified `Community`